recipes/pypardiso/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "pypardiso" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/haasad/PyPardisoProject/archive/refs/tags/v0.2.2.tar.gz
+  sha256: b1d3a139f8f04c74f8b2b5c4b55434e1f58b00ae3fe0ca975115971906d93eac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```

If my other commends are valid this looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

If there a cost to build a separate package for each version? We have already lost a decent amount of time debugging specific problems with versions and OSes, my preference would be to use the well-tested conda-forge architecture to build known good packages for each version and OS combination automatically.

---

Seems to build find with `noarch`